src/components/services-view/services-view.controller.ts

```diff
@@ -71,6 +71,15 @@ export class ServicesViewController implements angular.IController {
       }
     };
 
+    this.ctrl.noItemsConfig = {
+        title: 'No items.',
+        info: 'There are no catalog items. Please see:',
```

Comment:
To be consistent with other empty state text, I propose we say:
"No catalog items have been loaded."

And the helpLink:
"See Loading the Default Image Streams and Templates".

Reply:
Updated text and link